OktaJWT/APIErrors.swift

```diff
@@ -11,19 +11,19 @@
  */
 
 public enum OktaAPIError: Error {
-    case NoWellKnown
-    case NoJWKSEndpoint
-    case NoKey
+    case noWellKnown
```

Comment:
Any reason for the change to `lowerCamelCase`? Per the [Reference Manual](https://docs.swift.org/swift-book/ReferenceManual/Types.html), all Type Identifiers should be `UpperCamelCase`.

Also, it is possible that a user of this library is expecting these types to be `UpperCamelCase`, and this change will require them to update their application. If we decide to change the naming schema now, we're going to need to release a new major version of this library, as `2.0.0`.

Reply:
This is due to new guidelines proposal, which was accepted in new Swift:
https://github.com/apple/swift-evolution/blob/master/proposals/0006-apply-api-guidelines-to-the-standard-library.md
Look at "Lowercase enum cases and static properties" paragraph.

Also: https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html

By current Swift guidelines all enum cases should be lower camel case. So basically this change is due to moving to latest Swift requirements. It is unlikely that library users will have problems with it as in most cases Swift error enums are used as `error.localizedError`. For example `okta-sdk-appauth-ios`, which is main user of this library do not have anything to change after this update.

Reply:
I agree, let's plan on releasing this as version 2.0.0 because this is a breaking API change (despite being very small).

Reply:
Updated version to 2.0.0